### PR TITLE
Revert "Revert "Fix duplicate title tags development""

### PR DESCRIPTION
**Pull request by runleonarun**
Reverts dbt-labs/docs.getdbt.com#3324 and proposes new titles that were changed in https://github.com/dbt-labs/docs.getdbt.com/pull/3276

### CODE REVIEW

**Review by mirnawong1 — DISMISSED** [review]

File: website/docs/docs/build/exposures.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Exposures"
+title: "Build Your DAG - Exposures"
```
Comment (runleonarun) [inline website/docs/docs/build/exposures.md]:
```suggestion
title: "Exposures in your DAG"
sidebar_title: "Exposures"
```

File: website/docs/docs/build/metrics.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Metrics"
+title: "Build Your DAG - Metrics"
```
Comment (runleonarun) [inline website/docs/docs/build/metrics.md]:
```suggestion
title: "Metrics in your DAG"
sidebar_label: "Metrics"
```

File: website/docs/docs/build/exposures.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
-title: "Exposures"
+title: "Exposures in your DAG"
+sidebar_title: "Exposures"
```
Comment (runleonarun) [inline website/docs/docs/build/exposures.md]:
```suggestion
sidebar_label: "Exposures"
```

File: website/docs/docs/build/exposures.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
-title: "Exposures"
+title: "Exposures in your DAG"
```
Comment (runleonarun) [inline website/docs/docs/build/exposures.md]:
```suggestion
title: "Add Exposures to your DAG"
```

File: website/docs/docs/build/seeds.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Seeds"
+title: "Build Your DAG - Seeds"
```
Comment (runleonarun) [inline website/docs/docs/build/seeds.md]:
```suggestion
title: "Add Seeds to your DAG"
sidebar_label: "Seeds"
```

File: website/docs/docs/build/metrics.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
-title: "Metrics"
+title: "Metrics in your DAG"
```
Comment (runleonarun) [inline website/docs/docs/build/metrics.md]:
```suggestion
title: "Add metrics to your DAG"
```

File: website/docs/docs/build/snapshots.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Snapshots"
+title: "Build Your DAG - Snapshots"
```
Comment (runleonarun) [inline website/docs/docs/build/snapshots.md]:
```suggestion
title: "Add snapshots to your DAG"
sidebar_label: "Snapshots"
```

File: website/docs/docs/build/sources.md [hunk 8]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Sources"
+title: "Build Your DAG - Sources"
```
Comment (runleonarun) [inline website/docs/docs/build/sources.md]:
```suggestion
title: "Add sources to your DAG"
sidebar_label: "Sources"
```

File: website/docs/docs/build/tests.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Tests"
+title: "Enhance Your Models - Tests"
```
Comment (runleonarun) [inline website/docs/docs/build/tests.md]:
```suggestion
title: "Add tests to your DAG"
sidebar_title: "Tests"
```

File: website/docs/docs/dbt-cloud-apis/schema-metadata-snapshots.mdx [hunk 10]
```diff
@@ -1,5 +1,7 @@
 ---
-title: "Snapshots"
+title: "Schema - Snapshots"
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/schema-metadata-snapshots.mdx]:
```suggestion
```

**Review by mirnawong1 — DISMISSED** [review]

File: website/docs/reference/dbt-jinja-functions/debug-method.md [hunk 11]
```diff
@@ -1,5 +1,6 @@
 ---
-title: "debug"
+title: " About debug macro"
```
Comment (runleonarun) [inline website/docs/reference/dbt-jinja-functions/debug-method.md]:
```suggestion
title: "About debug macro"
```

File: website/docs/reference/dbt-jinja-functions/local-md5.md [hunk 12]
```diff
@@ -1,5 +1,6 @@
 ---
-title: "local_md5"
+title: "local_md5 context variable"
```
Comment (runleonarun) [inline website/docs/reference/dbt-jinja-functions/local-md5.md]:
```suggestion
title: "About local_md5 context variable"
```

**Comment by runleonarun** [comment]
Note: I opened a followup issue to [create better titles for the rest of the reference content](https://github.com/dbt-labs/docs.getdbt.com/issues/3334).

@JKarlavige I revised the titles to be more descriptive of their content and used sidebar_label to show the simple title.

File: website/docs/reference/artifacts/dbt-artifacts.md [hunk 13]
```diff
@@ -1,5 +1,6 @@
 ---
-title: Overview
+title: "About dbt Artifacts"
```
Comment (matthewshaver) [inline website/docs/reference/artifacts/dbt-artifacts.md]:
```suggestion
title: "About dbt artifacts"
```

File: website/docs/reference/commands/list.md [hunk 14]
```diff
@@ -1,5 +1,6 @@
 ---
-title: "ls (list)"
+title: "About dbt ls command"
```
Comment (matthewshaver) [inline website/docs/reference/commands/list.md]:
```suggestion
title: "About dbt ls (list) command"
```

File: website/docs/reference/dbt-jinja-functions/statement-blocks.md [hunk 15]
```diff
@@ -1,5 +1,6 @@
 ---
-title: "statement blocks"
+title: "About statement blocks"
+sidebar_label: "statement-blocks"
```
Comment (matthewshaver) [inline website/docs/reference/dbt-jinja-functions/statement-blocks.md]:
```suggestion
sidebar_label: "statement blocks"
```

**Review by matthewshaver — DISMISSED** [review]

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**